src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  color: $main-text-color;
+  background-color: #fff;
+  border: 1px solid $light-grey-color;
```

Comment:
Ensure that the SCSS variable `$light-grey-color` is defined and correctly imported in your SCSS setup to prevent any compilation errors.